.github/workflows/deploy-production.yml

```diff
@@ -0,0 +1,30 @@
+name: Deploy to production (aria-at.w3.org)
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy-production:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Install SSH key for deploying
+        uses: shimataro/ssh-key-action@v2
+        with:
+          key: ${{ secrets.DEPLOY_SSH_PRIVATE_KEY }}
+          known_hosts: ${{ secrets.DEPLOY_KNOWN_HOSTS_PRODUCTION }}
```

Comment:
This variable doesn't seem to be defined in GitHub! Is that expected?

Reply:
Added!